index.html

```diff
@@ -283,6 +283,28 @@ <h3>
     identified by the token "compute-pressure".
     Its [=policy-controlled feature/default allowlist=] is `["self"]`.
   </p>
+  <p>
+    Workers (dedicated and shared) adhere to the permission policy set by their
+    owning document(s).
+  </p>
+  <p>
+    Shared workers often have multiple owning documents as they can be obtained
+    by other documents with the [=same origin=].
+
```

Comment:
Did you mean to start another `<p>`aragraph here?

Reply:
No, just to make it easier for me to read in the editor